platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -4433,7 +4433,7 @@
         $scope.isFieldSelected = function(name) {
             var i;
             for (i = 0; i < $scope.selectedFields.length; i += 1) {
-                if ($scope.selectedFields[i].basic.name === name) {
+                if ($scope.selectedFields[i] !== undefined && $scope.selectedFields[i].basic.name === name) {
```

Comment:
Wouldn't "$scope.selectedFields[i] && $scope.selectedFields[i].basic.name === name" be enough?